src/index.html

```diff
@@ -9,6 +9,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
```

Comment:
<img width="122" alt="Screenshot 2023-08-10 at 17 07 30" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/73d39332-7921-4386-ade4-e7f98475eb1e">

delete underlink for nav__link